pipelines/youtube_data/__init__.py

```diff
@@ -0,0 +1,181 @@
+import datetime
+from typing import Any, Dict, Iterator, List, Sequence
+
+from googleapiclient.discovery import build
+
+import dlt
+from dlt.extract.source import DltResource, DltSource
+from dlt.common.typing import TDataItem
+
+
+_SERVICE_NAME = "youtube"
+_SERVICE_VERSION = "v3"
+current_time = datetime.datetime.now()
```

Comment:
The `current_time` is defined only once at the start of the script, which means that all videos will have the same `created_at` time. Is this something intentional? If not, consider moving the `current_time = datetime.datetime.now()` line into the `youtube_data` function to ensure it's accurate at the time of creating each data record.